### PR TITLE
New "Upgrade to v0.23 doc", changed existing Upgrade doc to be patch-specific

### PR DESCRIPTION
**Pull request by jwitz**
Resolves: https://github.com/astronomer/docs/issues/144

I took Steven's draft guide [here](https://github.com/astronomer/astronomer/blob/2337b526626ae6d9bb42a43a52a0bd4511891d42/bin/migration-scripts/lts-to-lts/0.16-to-0.23/README.md) and edited it for publication. Given that we have separate steps for patch/minor upgrades, I figured this was also a good time for us to make the guides more specific in their setup. The existing Upgrade Astronomer guide has been reworked into a patch-specific upgrade guide, while the new document will serve as a minor version-specific guide. 

If things look good here, I'll move the patch-specific doc to our 0.23 docset as well.

### CODE REVIEW

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
 ---
-title: "Enterprise Upgrade Guide"
-navTitle: "Upgrade Astronomer"
-description: "How to upgrade the Astronomer Enterprise Platform."
+title: "Upgrade to a Patch Version of Astronomer Enterprise"
+navTitle: "Upgrade to a Patch Version"
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:3]:
```suggestion
navTitle: "Upgrade to an Astronomer Patch Version"
```

Too long? I think not including it would cause some confusion with Airflow versioning, which could also be "patch"

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 2]
```diff
@@ -24,23 +28,11 @@ Astronomer platform releases follow a semantic versioning scheme. All versions a
...
-- The patch upgrade process will NOT affect running tasks if `upgradeDeployments.enabled=false`
-- Patch version updates will NOT cause any downtime to Astronomer services (UI, API, etc.)
-
-Read below for specific guidelines.
+For example, upgrading Astronomer from v0.16.4 to v0.16.5 would be considered upgrading to a patch version, whereas upgrading from v0.15.0 to v0.16.0 would be considered upgrading to the latest minor version.
 
-> **Note:** Astronomer v0.16.5 and beyond includes an improved upgrade process that allows Airflow Deployments to remain unaffected through a platform upgrade that includes changes to [the Airflow Chart](https://github.com/astronomer/airflow-chart).
->
-> Now, Airflow Chart changes only take effect when another restart event is triggered by a user (e.g. a code push, Environment Variable change, resource or executor adjustment, etc).
+The following setup is only for upgrading to the latest patch version. To upgrade to the latest minor version, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23)
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:33]: This feels a little too late in the game to clarify - should we give a clearer lay of the land in the ## Overview by bringing up the scope of this doc higher and then linking out to the minor upgrade doc sooner?

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+title: "Upgrade to Astronomer Enterprise v0.23"
+navTitle: "Upgrade to v0.23"
+description: "How to upgrade the Astronomer Enterprise Platform."
+---
+
+## Overview
+
+As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model.
+
+This guide walks through the process of upgrading your Astronomer Enterprise platform to v0.23, which is the latest minor version.
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:11]: Link out to 0.23 release notes? This will live in `https://www.astronomer.io/docs/enterprise/0.23/resources/release-notes` (next)

Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:11]:
```suggestion
This guide walks through the process of upgrading your Astronomer Enterprise platform from 0.16.x to v0.23, which is the latest minor version.
```

Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:11]: I would also add a note: 

> If you are running a version of Astronomer that's lower than [v0.16 (latest)](/docs/enterprise/v0.16/resources/release-notes),  submit a request to [Astronomer Support](https://support.astronomer.io). Our team will help you determine an upgrade path to Astronomer 0.16 first before you can upgrade to Astronomer v0.23.

(Sort of weirdly worded - but something like that)

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+```
+For example, all of the following images would work for this upgrade:
+```sh
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10-onbuild
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10
+astronomerinc/ap-airflow:1.10.5-9-buster-onbuild
+astronomerinc/ap-airflow:1.10.5-9-buster
+```
+> **Note:** While `-onbuild` and `<build-number>` are optional, we recommend including them for most upgrades. If you have your own build, test, and publish workflows that are layered on top of the Astronomer Airflow images, then removing `<build-number>` is appropriate.
+
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:42]: Spaces missing in the above section? E.g. after `format:` , before `For example`, and before "Note"

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+For example, upgrading Astronomer from v0.16.4 to v0.16.5 would be considered upgrading to a patch version, whereas upgrading from v0.15.0 to v0.16.0 would be considered upgrading to the latest minor version.
+
+The following setup is only for upgrading to the latest minor version. To upgrade to a patch version, read [Upgrade to a Patch Version of Astronomer Enterprise](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer)
+
+## Step 1: Check Version Compatibility
+
+Ensure that the following software
+
+- **Kubernetes**: Your version must be greater than or equal to 1.14 and less than 1.19. If you need to upgrade Kubernetes, contact your cloud provider's support or your Kubernetes administrator.
+- **Airflow Images**: You must be using an Astronomer Certified Airflow Image, and the version of your Image must be 1.10.5 or greater. In addition, your image should be in the following format:
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:30]: I'm actually not sure "Image" needs to be capitalized always - do you know? Double check?

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10-onbuild
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10
+astronomerinc/ap-airflow:1.10.5-9-buster-onbuild
+astronomerinc/ap-airflow:1.10.5-9-buster
+```
+> **Note:** While `-onbuild` and `<build-number>` are optional, we recommend including them for most upgrades. If you have your own build, test, and publish workflows that are layered on top of the Astronomer Airflow images, then removing `<build-number>` is appropriate.
+
+## Step 2: Check Permissions
+
+Minor version upgrades can be initiated only by Astronomer Admins. To confirm you're an Astronomer Admin, confirm that you have access to **System Admin** features in the in the Astronomer UI:
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:45]:
```suggestion
Minor version upgrades can be initiated only by a user with System Admin permissions on Astronomer. To confirm you're an Astronomer SysAdmin, confirm that you have access to **System Admin** features in the in the Astronomer UI:
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+$ kubectl auth can-i create sa --namespace <your-astronomer-namespace>
+$ kubectl auth can-i create jobs --namespace <your-astronomer-namespace>
+```
+
+If all commands return `yes`, then you have the appropriate Kubernetes permissions.
+
+## Step 3: Backup Your Database
+
+Backup your database using your cloud provider's functionality for doing so, or make a request to your database administrator to backup based on your organization's own guidelines.
+
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:62]: Probably a question for @sjmiller609 - do we want/need to give users more info here? Should we link out to our Disaster Recovery doc, which talks about this in more detail?

Should we also clarify "what database"? Assuming here we're talking about Platform DB

**Review by paolaperaza — COMMENTED** [review]
Some initial comments, @jwitz ! I'll finish reviewing later this afternoon.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+## Step 4: Check the Status of Your Kubernetes Pods
+
+All pods should be either `Running` or `Completed`. If you have any pods that are crashing, but this is expected behavior and you want to proceed anyways, make note of which pods are crashing before upgrading.
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:65]:
```suggestion
All pods should be either `Running` or `Completed`, though some may be in `CrashLoopBackOff`. If you have any pods that are crashing, make note of which ones before upgrading.
```

A little confused on the recommendation here given the first draft of this - should users proceed? Or is it fine as long as they make note of them now?

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+```sh
+$ kubectl config set-context --current --namespace=default
+```
+
+## Step 6: Upgrade Astronomer
+
+Run the following command to begin the upgrade process:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/astronomer/astronomer/master/bin/migration-scripts/lts-to-lts/0.16-to-0.23/manifests/upgrade.yaml
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:80]:
```suggestion
$ kubectl apply -f https://raw.githubusercontent.com/astronomer/astronomer/master/bin/migration-scripts/lts-to-lts/0.16-to-0.23/manifests/upgrade.yaml
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+While your platform is upgrading, monitor your pods to ensure that no errors occur. To do so, first find the names of your pods by running the following command:
+
+```sh
+$ kubectl get pods | grep upgrade-astronomer
+```
+
+Then, run the following command to see each pod while you upgrade:
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:89]:
```suggestion
Then, run the following command to follow the logs of any pod while you upgrade:
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+$ kubectl logs <your-pod-name>
+```
+
+## Step 7: Confirm That the Upgrade Was Successful
+
+If the upgrade was successful, you should be able to:
+
+* Log in to Astronomer at `astronomer.BASEDOMAIN`
+* See Workspaces and Airflow Deployments in the Astronomer UI
+* Access the settings of your Airflow Deployments
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:101]: What are the "settings of your Airflow Deployments"? Is that the **Settings** tab in the Astronomer UI?

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+* See metrics on the **Metrics** tab in the Astronomer UI
+* Successfully run `$ astro deploy` using the Astronomer CLI
+* Open a Deployment in the Airflow UI
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:104]:
```suggestion
* Open the Airflow UI of any Deployment
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+* Access logs for your DAGs in the Airflow UI
+
+## Roll Back to Enterprise v0.16
+
+If the upgrade has some issues and you need to recover your platform, you can roll back to v0.16. To do so:
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:109]:
```suggestion
If you encounter an issue during your upgrade such that you'd like to recover the platform, you may choose to roll back to Astronomer v0.16. To do so:
```

**Review by paolaperaza — CHANGES_REQUESTED** [review]
A few comments in here @jwitz - let me know what you think on these and I'll give it another look!

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md (hunk 1 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:3]: I think this being in the `Manage Astronomer` tab would resolve any confusion on that front.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 5 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:22]: Kubernetes does not capitalize it, so that seems like the move!

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 4 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:34]: Because this is a bulleted list, we need to keep the spacing tight so the notes on Airflow images appear under the Airflow images bullet.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 8 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:57]: I think the intention here was to ensure that readers have a way of knowing which pods are crashing because of the upgrade, vs which ones are crashing for other reasons. I tried clarifying that they can, in fact, proceed if they have crashing pods.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 11 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:93]: I think so, but @stevenjs1 can confirm.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 13 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:101]: Updated this wording slightly in my latest commit.

**Comment by jwitz** [comment]
Thanks for the edits @paolaperaza! Left some replies and made changes based on your suggestions.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+---
+
+## Overview
+
+As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model.
+
+This guide walks through the process of upgrading your Astronomer Enterprise platform from 0.16.x to [v0.23](https://www.astronomer.io/docs/enterprise/0.23/resources/release-notes), which is the latest minor version.
+
+A few notes before you start:
+- You must be on Astronomer Enterprise v0.16.x in order to complete this setup. If you are running a version of Astronomer that's lower than v0.16, submit a request to Astronomer Support. Our team will help you upgrade from your earlier version to v0.23.
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:14]:
```suggestion
- You must be on Astronomer Enterprise v0.16.x in order to upgrade to Astronomer 0.23+. If you are running a version of Astronomer that's lower than v0.16, submit a request to [Astronomer Support](https://support.astronomer.io) and our team will help you define an alternate upgrade path.
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+title: "Upgrade to Astronomer Enterprise v0.23"
+navTitle: "Upgrade to v0.23"
+description: "How to upgrade the Astronomer Enterprise Platform."
+---
+
+## Overview
+
+As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model.
+
+This guide walks through the process of upgrading your Astronomer Enterprise platform from 0.16.x to [v0.23](https://www.astronomer.io/docs/enterprise/0.23/resources/release-notes), which is the latest minor version.
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:11]:
```suggestion
This guide walks through the process of upgrading your Astronomer Enterprise platform from 0.16.x to [v0.23](https://www.astronomer.io/docs/enterprise/0.23/resources/release-notes), which is the latest available LTS release.
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+
+A few notes before you start:
+- You must be on Astronomer Enterprise v0.16.x in order to complete this setup. If you are running a version of Astronomer that's lower than v0.16, submit a request to Astronomer Support. Our team will help you upgrade from your earlier version to v0.23.
+- The following setup is only for upgrading to the latest minor version. To upgrade to the latest patch version, read [Upgrade to a Patch Version of Astronomer Enterprise](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer).
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:15]:
```suggestion
- The guidelines below only apply to users who are upgrading to the Astronomer v0.23 series for the first time. Once you've completed the upgrade to any v0.23 version, you'll be free to upgrade to subsequent v0.23.x patch versions as they are released by our team. For instructions, read [Upgrade to a Patch Version of Astronomer Enterprise](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer).
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 8 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md]: Got it hm, I think the wording is mixing me up here. Made another suggestion!

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+
+If all commands return `yes`, then you have the appropriate Kubernetes permissions.
+
+## Step 3: Backup Your Database
+
+Backup your database using your cloud provider's functionality for doing so, or make a request to your database administrator to backup based on your organization's own guidelines.
+
+## Step 4: Check the Status of Your Kubernetes Pods
+
+All pods should be either `Running`, `Completed`, or in `CrashLoopBackOff`. If you have any pods that are crashing, but this is expected behavior and you want to proceed anyways, make note of which ones before before proceeding.
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:57]: I'd still rephrase this to say:

> Before you proceed with the upgrade, ensure that the Kubernetes Pods in your platform namespace are healthy. To do so, run:
> 
> ```
> $ kubectl get pods -n <your-astronomer-namespace>
> ```
> 
> All pods should be in either the `Running` or `Completed` state. If any of your pods are in a `CrashLoopBackOff` state or are otherwise unhealthy, make sure that's expected behavior before you proceed.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+* Access logs for your DAGs in the Airflow UI.
+
+## Roll Back to Enterprise v0.16
+
+If you encounter an issue during your upgrade that requires you to recover your original platform, you can roll back to Astronomer v0.16. To do so:
+
+1. Apply the rollback automation script by running the following command:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/astronomer/astronomer/master/bin/migration-scripts/lts-to-lts/0.16-to-0.23/manifests/rollback.yaml
+```
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:106]: @sjmiller609 Can you shed more insight on what this script calls for? Are Metadata Databases for all Airflow Deployments preserved?

**Review by paolaperaza — COMMENTED** [review]
Made some more comments, @jwitz but we're close!

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 19]
```diff
@@ -1,20 +1,23 @@
 ---
-title: "Enterprise Upgrade Guide"
-navTitle: "Upgrade Astronomer"
-description: "How to upgrade the Astronomer Enterprise Platform."
+title: "Upgrade to a Patch Version of Astronomer Enterprise"
+navTitle: "Upgrade to a Patch Version"
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:3]:
```suggestion
title: "Update the patch version of Astronomer Enterprise"
navTitle: "Update patch version"
```

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 20]
```diff
@@ -1,20 +1,23 @@
...
+description: "How to upgrade your Astronomer Enterprise Platform to a new patch version."
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:4]:
```suggestion
description: "How to update your Astronomer Enterprise Platform to a new patch version."
```

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 21]
```diff
@@ -1,20 +1,23 @@
...
 ---
 
 ## Overview
 
-As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model. Critical security and bug fixes will be regularly shipped as patch versions that follow LTS releases. Patch releases will be made available _between_ quarterly LTS releases and require a simple upgrade process.
+As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model. Critical security and bug fixes will be regularly shipped as patch versions that follow LTS releases.
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:9]: took longer than a quarter, so maybe run this by engineering to make sure quarterly vs bi-quarterly

Comment (sjmiller609) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:9]: Also, it's important that we say how long an LTS version will be supported, I think we should say 1 year.

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md [hunk 22]
```diff
@@ -24,23 +27,11 @@ Astronomer platform releases follow a semantic versioning scheme. All versions a
 - Y: Minor Version
 - Z: Patch/Hotfix
 
-For example, upgrading Astronomer from v0.16.4 to v0.16.5 would be considered upgrading to a "patch" version whereas upgrading from v0.15.0 to v0.16.0 would be considered upgrading to the latest "minor" version.
-
-## Upgrade to a Patch Version
+For example, upgrading Astronomer from v0.16.4 to v0.16.5 would be considered upgrading to a patch version, whereas upgrading from v0.15.0 to v0.16.0 would be considered upgrading to the latest minor version.
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:30]:
```suggestion
For example, upgrading Astronomer from v0.16.4 to v0.16.5 would be considered upgrading a patch version, whereas upgrading from v0.15.0 to v0.16.0 would be considered upgrading a minor version.
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 23]
```diff
@@ -0,0 +1,113 @@
+---
+title: "Upgrade to Astronomer Enterprise v0.23"
+navTitle: "Upgrade to v0.23"
+description: "How to upgrade the Astronomer Enterprise Platform."
+---
+
+## Overview
+
+As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model.
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:9]: Make sure we decided quarterly, and specify how long they are supported for. e.g. if we do quarterly and yearly support, then we will need to support 4 at a time. I think we might consider bi quarterly with yearly support.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 24]
```diff
@@ -0,0 +1,113 @@
...
+astronomerinc/ap-airflow:<airflow-version>-<build-number>-<distribution>-onbuild
+```
+For example, all of the following images would work for this upgrade:
+```sh
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10-onbuild
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10
+astronomerinc/ap-airflow:1.10.5-9-buster-onbuild
+astronomerinc/ap-airflow:1.10.5-9-buster
+```
+> **Note:** While `-onbuild` and `<build-number>` are optional, we recommend including them for most upgrades. If you have your own build, test, and publish workflows that are layered on top of the Astronomer Airflow images, then removing `<build-number>` is appropriate.
```
Comment (sjmiller609) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:33]:
```suggestion
> **Note:** While `-onbuild` and `<build-number>` are optional, we recommend including them for most upgrades. If you have your own build, test, and publish workflows that are layered on top of the Astronomer Airflow images, then removing `<build-number>` is appropriate. The images including build number are immutable.
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 7 above)
Comment (sjmiller609) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:54]: the whole database instance, i.e. backing up everything

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 18 above)
Comment (sjmiller609) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:106]: it restores the platform DB and the helm state of the Astronomer helm chart

**Review by sjmiller609 — APPROVED** [review]

**Review by krisdock — APPROVED** [review]
my concerns with clarity on the default namespace stuff are already addressed here. lgtm.

https://github.com/astronomer/docs/pull/200/files#diff-06af45c2f0d7e0f92d6c849a8a7804b2cc2d33529ed401a2138fef4843a3aad1R59

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md (hunk 1 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:3]: @jwitz Fair - I think the nav title would be too long. Let's keep yours!

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 7 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:54]: So maye -

```suggestion
Backup your entire Astronomer database instance using your cloud provider's functionality for doing so, or make a backup request to your database administrator based on your organization's guidelines.
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 11 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md]: @jwitz Maybe then rephrase to: `Access the **Settings** tab of all Airflow Deployments`

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md (hunk 21 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:9]: @sjmiller609 Yea, we should clarify that here. @jwitz Let's keep moving on this PR and let me chat with @astrorfox about how we want to communicate EOL on v0.16 + v0.23 - we can follow up on this.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 25]
```diff
@@ -0,0 +1,113 @@
...
+
+## Roll Back to Enterprise v0.16
+
+If you encounter an issue during your upgrade that requires you to recover your original platform, you can roll back to Astronomer v0.16. To do so:
+
+1. Apply the rollback automation script by running the following command:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/astronomer/astronomer/master/bin/migration-scripts/lts-to-lts/0.16-to-0.23/manifests/rollback.yaml
+```
+
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:107]:
```suggestion
This restores the Platform Database and the helm state of the Astronomer helm chart.

```

**Review by paolaperaza — APPROVED** [review]
Looks good @jwitz ! Still a few comments from Steven and me, but nothing major - will let you take care of those and merge as needed.

One followup note I made: Let's chat with Ryan/others on the team to figure out how we want to communicate EOL for v0.16 + v0.23. That probably won't get figured out in the next few mins, but let's followup on that and add to our docs within the next few days.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 17 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:57]: Sounds good, was not quite understanding the `CrashLoopBackOff` part of things before, but this clears it up. Updated!

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md (hunk 19 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:3]: Based on some docsets from other companies, I think the current wording is appropriate. When people are looking for these docs, I think they're looking to upgrade Astronomer rather than updating a version.

File: enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md (hunk 22 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/05_upgrade-astronomer.md:30]: Based on some docsets from other companies, I think the current wording is appropriate. When people are looking for these docs, I think they're looking to upgrade Astronomer rather than updating a version.

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 25 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:107]: Took the capitalization out of "Platform Database", unless that's an official Astronomer term?
